conf/helmfile.d/0110.prometheus-redis-exporter.yaml

```diff
@@ -9,34 +9,31 @@ helmDefaults:
 
 repositories:
  # Stable repo of official helm charts
-  - name: "stable"
-    url: "https://kubernetes-charts.storage.googleapis.com"
+  - name: stable
+    url: https://kubernetes-charts.storage.googleapis.com
 
 releases:
-  - name: "prometheus-redis-exporter"
+  - name: prometheus-redis-exporter
     labels:
-      chart: "prometheus-redis-exporter"
-    chart: "stable/prometheus-redis-exporter"
-    version: "3.2.1"
-    namespace: "monitoring"
+      chart: prometheus-redis-exporter
+      component: monitoring
+      namespace: monitoring
+    chart: stable/prometheus-redis-exporter
+    version: 3.3.3
+    namespace: monitoring
     values:
-      - resources:
-          requests:
-            cpu: 100m
-            memory: 100Mi
-
-        script:
+      - script:
           configmap: prometheus-redis-exporter-script
           keyname: script
 
         service:
           annotations:
-            prometheus.io/path: /metrics
+            prometheus.io/path: "/metrics"
             prometheus.io/port: "9121"
             prometheus.io/scrape: "true"
 
         annotations:
-          prometheus.io/path: /metrics
+          prometheus.io/path: "/metrics"
```

Comment:
Just for my own edification: what's the rule for when we do and don't need quotes?

Reply:
annotations always need to be strings.

Otherwise, there is an issue with using a `False` value or `0` without strings, as the templating sees the `False` and thinks it should not pass it through.